__tests__/sequencerProvider.test.ts

```diff
@@ -19,7 +20,7 @@ describeIfSequencer('SequencerProvider', () => {
     sequencerProvider = getTestProvider() as SequencerProvider;
     customSequencerProvider = new Provider({
       sequencer: {
-        baseUrl: 'https://alpha4.starknet.io',
+        baseUrl: 'http://127.0.0.1:5050/',
```

Comment:
This change was made to speed up testing while also showcasing "customSequencerProvider"